src/test/resources/feature/2-mittente/1_invioNotifiche/002_001_neg_iniviNotificaSezioneInformazioniPreliminariDatiErrati.feature

```diff
@@ -22,4 +22,7 @@ Feature: Il mittente inserisce i dati non coretti nella sezione informazioni pre
     And Nella section si visualizza il popup vuoi uscire
     And Nella section cliccare sul tasto esci
     Then Si visualizza correttamente la pagina Piattaforma Notifiche
-    And Logout da portale mittente
\ No newline at end of file
+    And Nella pagina Piattaforma Notifiche cliccare sul bottone Invia una nuova notifica
+    And Si visualizza correttamente la pagina Piattaforma Notifiche section Informazioni preliminari
+    And Nella section si prova ad cliccare sul tasto continua senza aver inserito nessun dato
```

Comment:
questo step non controlla se i campi siano vuoti ma sul tanto che il tasto continua sia disabilitato

Reply:
No basta sul tanto che uno dei campi obbligatori sia vuoto il tasto continua  rimarrà disabilitato

Reply:
Done, now there is a method that check if the field is empty